lib/localize.js

```diff
@@ -258,7 +265,9 @@ export const Localize = class extends getLocalizeClass() {
 	constructor(config) {
 		super();
 		super.constructor.setLocalizeMarkup(localizeMarkup);
-		this.localize = (...args) => super.localize(...args);
+		this.localize = super.constructor.documentLocaleSettings.pseudoLocalization?.isAvailable
+			? (...args) => super.constructor.pseudoLocalize((...args) => super.localize(...args), ...args)
```

Comment:
Somewhat surprisingly, this is appreciably faster than various other ways to bind `this` and pass the arguments through.